src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/PaymentAccountValidator.java

```diff
@@ -77,4 +78,27 @@ public void checkPbasAreUniqueWithOrgId(Set<String> paymentAccounts, String orgI
         }
     }
 
+    public boolean checkSinglePbaIsValid(String pbaAccount) {
+        if (pbaAccount.length() == 10) {
```

Comment:
This method is already present in existing code. Please reuse.

Reply:
removed method